scripts/app.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+//Constructor Function
+var Portfolio = function (portfolio) {
+  this.title = title;
```

Comment:
You probably want this as a function, not var `function Portfolio(portfolio) { ...`